utils/display.go

```diff
@@ -51,3 +54,35 @@ func DisplayJsonOutput(output any, out io.Writer) error {
 	}
 	return nil
 }
+
+func DisplayMessage(message string, messageType string, indented bool, out io.Writer) {
+	prefix := "==>"
+	if indented {
+		prefix = "  ->"
+	}
+
+	if messageType == "info" && !indented {
+		msg := text.Bold.Sprintf("%s %s\n", prefix, message)
+		fmt.Fprint(out, msg)
+		return
+	}
+
+	prefixTpl := ""
+	if slices.Contains([]string{"error", "warning", "success", "info"}, messageType) {
+		prefixTpl = colorizeMessage(
+			fmt.Sprintf("%s %s: ", prefix, strings.ToUpper(messageType)),
+			messageType,
+		)
+	}
+	fmt.Fprintf(out, "%s%s\n", prefixTpl, message)
+}
+
+func colorizeMessage(msg string, msgType string) string {
+	colors := map[string]text.Color{
+		"success": text.FgGreen,
+		"warning": text.FgYellow,
+		"error":   text.FgRed,
+		"info":    text.FgCyan,
```

Comment:
(minor) What about using an enum for the message type, both here and in `DisplayMessage`? Go doesn't really have enums, but something similar can be done like this: https://github.com/jedib0t/go-pretty/blob/eb654fb730581a1a87290d9b1087867232a7b314/text/color.go#L33-L48

Reply:
Tried to quickly implement something, but it looked a bit messy, since we have to define an enum and still a map to match the values

Reply:
What I wanted to achieve was to avoid having to use strings to call the function, that is having to pass `"success"` or one of the other options, as it is quite error prone (e.g. typos that go unnoticed). Maybe we can at least define them as constants?

Something like this:
```go
type MessageType string

const (
    Success MessageType = "success"
    ...
)
```

Inside the functions we can just convert them back to strings, so that we do not have to change the functions' code. What do you think? Are there any other cons I am missing?

Reply:
Thanks for the feedback, I think I managed to find a nice solution, please have a look

Reply:
I think this solution is great!